src/Compilers/CSharp/Test/Semantic/Semantics/NullCheckedParameterTests.cs

```diff
@@ -129,59 +129,49 @@ public void NullCheckedBadSyntax()
 #pragma warning disable CS8893  
```

Comment:
>warning disable CS8893 [](http://example.com/codeflow?start=8&length=22)

Was this `#pragma` disabling the warning previously? Does it now? (Was there a reason to change the parameter default value from `null` in each of the methods?)

Reply:
*sigh* yes, some renumbering must have broken it. I feel it's simpler to use a literal string here, so if you don't object I'd like to keep it as it is in this PR. But I will delete the pragma.